head.go

```diff
@@ -59,7 +59,8 @@ type Head struct {
 	appendPool sync.Pool
 	bytesPool  sync.Pool
 
-	minTime, maxTime int64
+	minTime, maxTime int64 // Current min and max of the samples included in the head.
+	minValidTime     int64 // Mint allowed to be added to te head. It shouldn't be lower than the last persisted block.
```

Comment:
`te head` --> `the head`

Reply:
`It shouldn't be lower than the last persisted block` --> `It shouldn't be lower than the max time of the last persisted block`

Reply:
Also, I think we should be updating `h.minValidTime` in `h.Truncate` it doesn't make sense to accept samples that fall in the truncated period.

Reply:
yep logical!, thanks, updated!